splicemachine/spark/context.py

```diff
@@ -92,6 +106,15 @@ def tableExists(self, schema_table_name):
         """
         return self.context.tableExists(schema_table_name)
 
+    def tableExists2(self, schema_name, table_name):
```

Comment:
I think we should combine these two functions so users can pass in either to a single tableExists

Reply:
Updated the function in new commit [6ba47bc](https://github.com/splicemachine/pysplice/pull/54/commits/6ba47bcab47845eff66334f14949069862aa25ac).